ios/RNZendeskChat.xcodeproj/project.pbxproj

```diff
@@ -42,9 +45,11 @@
 		B6462EB61C603E340010294B = {
 			isa = PBXGroup;
 			children = (
+				A73FC0C32550745000B7EC05 /* jwtAuth.swift */,
```

Comment:
Adding Swift to react-native libraries can cause problems for some customers. Was this the only way to accomplish the task?